src/include/duckdb/main/capi/header_generation/apis/extension_api_v0.json

```diff
@@ -0,0 +1,299 @@
+                {"name": "duckdb_appender_error"},
+                {"name": "duckdb_appender_flush"},
+                {"name": "duckdb_appender_close"},
+                {"name": "duckdb_appender_destroy"},
+                {"name": "duckdb_appender_begin_row"},
+                {"name": "duckdb_appender_end_row"},
+                {"name": "duckdb_append_default"},
+                {"name": "duckdb_append_bool"},
+                {"name": "duckdb_append_int8"},
+                {"name": "duckdb_append_int16"},
+                {"name": "duckdb_append_int32"},
+                {"name": "duckdb_append_int64"},
+                {"name": "duckdb_append_hugeint"},
+                {"name": "duckdb_append_uint8"},
+                {"name": "duckdb_append_uint16"},
+                {"name": "duckdb_append_uint32"},
+                {"name": "duckdb_append_uint64"},
+                {"name": "duckdb_append_uhugeint"},
+                {"name": "duckdb_append_float"},
+                {"name": "duckdb_append_double"},
+                {"name": "duckdb_append_date"},
+                {"name": "duckdb_append_time"},
+                {"name": "duckdb_append_timestamp"},
+                {"name": "duckdb_append_interval"},
+                {"name": "duckdb_append_varchar"},
+                {"name": "duckdb_append_varchar_length"},
+                {"name": "duckdb_append_blob"},
+                {"name": "duckdb_append_null"},
+                {"name": "duckdb_append_data_chunk"},
+                {"name": "duckdb_query_arrow"},
+                {"name": "duckdb_query_arrow_schema"},
+                {"name": "duckdb_prepared_arrow_schema"},
+                {"name": "duckdb_result_arrow_array"},
+                {"name": "duckdb_query_arrow_array"},
+                {"name": "duckdb_arrow_column_count"},
+                {"name": "duckdb_arrow_row_count"},
+                {"name": "duckdb_arrow_rows_changed"},
+                {"name": "duckdb_query_arrow_error"},
+                {"name": "duckdb_destroy_arrow"},
+                {"name": "duckdb_destroy_arrow_stream"},
+                {"name": "duckdb_execute_prepared_arrow"},
+                {"name": "duckdb_arrow_scan"},
+                {"name": "duckdb_arrow_array_scan"},
+                {"name": "duckdb_execute_tasks"},
+                {"name": "duckdb_create_task_state"},
+                {"name": "duckdb_execute_tasks_state"},
+                {"name": "duckdb_execute_n_tasks_state"},
+                {"name": "duckdb_finish_execution"},
+                {"name": "duckdb_task_state_is_finished"},
+                {"name": "duckdb_destroy_task_state"},
+                {"name": "duckdb_execution_is_finished"},
+                {"name": "duckdb_stream_fetch_chunk"}
+            ]
+        },
+        {
+            "version": "v0.0.2",
+            "entries": [
+                { "name": "duckdb_open_ext" }
+            ]
+        },
+        {
+            "version": "v0.0.3",
```

Comment:
"version": "dev"

This is the unreleased version that contains all added structs

Before a release we modify this to `"version": "x"` and empty the dev version